service/src/test/java/de/adorsys/psd2/sandbox/xs2a/pis/PIS.feature

```diff
@@ -29,7 +29,7 @@ Feature: PIS
     Then the transaction status <status> is received
     Examples:
       | payment-type | iban                   | payment-product       | psu-id         | password | sca-method | tan   | status |
-      | single       | DE11760365688833114935 | sepa-credit-transfers | PSU-Successful | 12345    | SMS_OTP    | 54321 | ACSC   |
+      | single       | DE11760365688833114935 | sepa-credit-transfers | PSU-Successful | 12345    | SMS_OTP    | 54321 | ACSP   |
```

Comment:
It would make sense to add two more tests. One for future-dated and one for periodic payments with status: ACTC

Reply:
done